Directory.Build.props

```diff
@@ -45,6 +45,10 @@
 
   <!-- Manually import the Versions.props file when the Arcade SDK isn't used. -->
   <Import Project="$(RepositoryEngineeringDir)Versions.props" Condition="'$(SkipArcadeSdkImport)' == 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)VmrLayout.props" />
+  <!-- For source only builds, override the version props with previously built artifacts.
+       If a version should be pinned, declare it after this import. -->
```

Comment:
I feel like this sentence could be removed as it provides less value with this refactor and there is very little chance this should ever be used.